test/aleph/http/core_test.clj

```diff
@@ -1,11 +1,9 @@
 (ns aleph.http.core-test
-  (:use
-    [clojure test])
   (:require
-    [aleph.http.core :as core])
+   [aleph.http.core :as core]
+   [clojure.test :refer [deftest is]])
```

Comment:
As you can see, I changed the indentation of these forms. That's because I used https://github.com/clojure-emacs/refactor-nrepl's `clean-ns` refactoring which produces this indentation by default. I did a quick research on how to change this but couldn't come up with anything so far. Let me know if you mind, otherwise I'll make it work *somehow* :smile:

Reply:
FTR: I just figured out how to adapt to it! Turns out that this is actually not `refactor-nrepl`'s job bur rather `clojure-mode`'s. So yeah, let me know if you'd like to preserve this!

Reply:
The new 1-space indentation is more standard, according to https://guide.clojure.style/#line-break-ns-declaration.

We should probably make a single reformat-only PR. Otherwise the indentation either never gets fixed, or it gets fixed bit-by-bit and clutters up PR diffs with formatting changes.

Reply:
Alright, 1 space it is. If you don't mind, I would already apply this for the test suite in this PR. The next one will then do the same for the main code.